src/qtism/runtime/tests/AssessmentTestSession.php

```diff
@@ -677,7 +698,7 @@ public function mustTrackPath()
      */
     public function mustAlwaysAllowJumps()
     {
-        return (bool)($this->getConfig() & self::ALWAYS_ALLOW_JUMPS);
+        return (bool)($this->getConfig() & self::ALWAYS_ALLOW_JUMPS) || $this->alwaysAllowJumps;
```

Comment:
The first condition is a bit heavier. I recommend putting it to the end of the logical `or`.